pkgs/by-name/st/steam/package.nix

```diff
@@ -45,23 +45,157 @@ let
       '')
     ] ++ extraPkgs pkgs;
 
-    # https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/blob/main/docs/distro-assumptions.md#shared-libraries
     multiPkgs = pkgs: with pkgs; [
```

Comment:
The trouble is that the new shorter list relies on including the closures which is what's at least partially broken.

We could go and manually sort out the tree of deps for the minimal packages but that's error prone and this will be reverted anyways sometime in the 25.05 cycle when we're reasonably sure that includeClosures works as expected.